packages/x-date-pickers/src/tests/describeValidation/testDayViewValidation.tsx

```diff
@@ -181,44 +181,6 @@ export const testDayViewValidation: DescribeValidationTestSuite = (ElementToTest
       expect(screen.getByLabelText('Next month')).to.have.attribute('disabled');
     });
 
-    it('should apply minTime', function test() {
```

Comment:
Those two tests were only run on the calendar and it now fire an error because `minTime` and `maxTime` is passed to the DOM